api/src/Entity/User.php

```diff
@@ -194,6 +206,13 @@ public function __construct() {
         $this->collaborations = new ArrayCollection();
```

Comment:
Mir ist noch aufgefallen dass `$plainPassword` in der `read` Group ist. Hat keine Auswirkungen, da `readable: false`, aber sollten wir vielleicht dennoch entfernen, ist sonst nur verwirrend.

Reply:
done